gapis/api/transform/transform_chain.go

```diff
@@ -118,12 +125,17 @@ func (chain *TransformChain) endChain(ctx context.Context) error {
 }
 
 func (chain *TransformChain) transformCommands(ctx context.Context, id CommandID, inputCmds []api.Cmd, beginTransformIndex int) error {
+
+	cmds := inputCmds
```

Comment:
I added this variable because as the code was before "inputCmds" contained the input commands at the beginning of the function, but was reassigned half way though so that it no longer contained the input commands. This seemed like bad practice to me. A variable's name should describe what value it stores!